sbt-plugin-test/build.sbt

```diff
@@ -75,17 +75,21 @@ lazy val testFrameworkJS = testFramework.js
 lazy val testFrameworkJVM = testFramework.jvm
 
 lazy val multiTest = crossProject.
+  enablePlugins(ScalaJSJUnitPlugin).
```

Comment:
Woops! I almost missed that! This will add the `ScalaJSJUnitPlugin` (and by transitivity, the `ScalaJSPlugin`) on *both* the JVM and JS parts of the crossProject, effectively turning the JVM part into a Scala.js project as well ^^

You should instead write:
```scala
  jsConfigure(_.enablePlugins(ScalaJSJUnitPlugin)).
```